website/src/pages/analyzer/index.mdx

```diff
@@ -74,8 +72,8 @@ biome check --apply-unsafe ./path/to/src
 
 ### Import sorting via VSCode extension
 
-
-The Biome VS Code extension has experimental support for imports sorting through the "Organize Imports" code action. By default this action can be run using the <kbd title="Shift">⇧</kbd>+<kbd>Alt</kbd>+<kbd>O</kbd> keyboard shortcut, or is accessible through the _Command Palette_ (<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd title="Shift">⇧</kbd>+<kbd>P</kbd>) by selecting _Organize Imports_.
+The Biome VS Code extension has experimental support for imports sorting through the "Organize Imports" code action.
```

Comment:
I believe we can remove the "experimental" part. Now the import sorting is enabled by default